requirements/docs.in

```diff
@@ -0,0 +1,43 @@
+alabaster==0.7.12
```

Comment:
I thought the idea of the *.in files was that they only contained the top level requirements and pip-compile handled pinning all the sub-deps in the *.txt files? This looks to be a list of all of the deps and sub-deps.

Reply:
Yeah, the *.in files are cirrently based on the manually handled reqs files which had everything in them. 

I've teased out the a few subdeps, and can do more - just a question of whether we want to land it all at once, or now + follow up